src/Containers/FeaturedProfilesCard.tsx

```diff
@@ -21,140 +22,118 @@ export interface IFeaturedProfilesCardProps {
     alt: string;
     width?: number;
     height?: number;
-    onClick?: Function;
 }
 
 export const FeaturedProfilesCard: React.FC<IFeaturedProfilesCardProps> = ({
-    onClick = (): void => {
-        return undefined;
-    },
     alt,
     profileData,
     width = 100,
     height = 100,
 }): React.ReactElement => {
     const getProfileCircles = useCallback(() => {
+        const getProfilePictureLimit = () => {
```

Comment:
Most of the naming here is great, little nitpick. Get is usually associated with making an request or fetching some data. Maybe renderProfileCircles is more appropriate.